### PR TITLE
Added contact us page.

### LINKED ISSUE #24: Contact Page

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #24

### CODE REVIEW

**Comment by terrellt** [comment]
Gotta rebase now.

File: spec/features/contact_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  context "when there are settings configured" do
+    let(:setting){create(:setting, :setting_name => 'Contact', :value => "Yo!")}
+    before do
+      setting
+    end
+    context "and the contact page is visited" do
+      before do
+        click_link "Contact Us"
+      end
+      it 'should say there are no settings configured' do
```
Comment (balesh2) [inline spec/features/contact_spec.rb]: it "should display the settings"

Comment (terrellt) [inline spec/features/contact_spec.rb]: @straleyb Can you look at this note?

Comment (straleyb) [inline spec/features/contact_spec.rb]: Definitely! Ill get that fixed asap.

**Comment by balesh2** [comment]
:+1:

**Pull request merged**